app/assets/javascripts/pages/vacancies/new/new.coffee

```diff
@@ -0,0 +1,14 @@
+App.Vacancies.New =
+  bind_fields: ->
+    $('#new_vacancy input').keypress (e) ->
+      if e.which == 13
+        $('#new_vacancy').submit()
+        return false
+
+$(document).on "turbolinks:load", ->
+  #TODO monkeypatch to allow the correct behave with summernote
+  # Remove when it will be fixed on summernote side
+
+  return unless $(".vacancies.new").length > 0 ||
+    $(".vacancies.create").length > 0
```

Comment:
This logic is only for the `new` or `create` (rerendered) screen. Not all vacancies (it is only to fix the summernote behave). But I think we can move it to Summernote componente. What you think @lastk ?